xtext-gradle-plugin/src/main/java/org/xtext/gradle/tasks/XtextExtension.xtend

```diff
@@ -146,7 +150,7 @@ class GeneratorConfig {
 		action.execute(outlets)
 	}
 
-	def getOutlet() {
+	@Internal def getOutlet() {
```

Comment:
without this gradle complains. => do i see it correct that all `getXXXX` methods are investigated?

Reply:
Yes, that's correct.